35628 and others - MULTIPLE tickets referred to in this Pull Request

tyndmar opened the pull request:
see ticket numbers
35596
25507
35628
35624
34931
25458

Commit:
35628 and others - MULTIPLE tickets referred to in this Pull Request
see ticket numbers
35596
25507
35628
35624
34931
25458

brad-advantzware reviewed (commented):
I see references to N:\ in the code.  There are too many changes to review without discussion.  Let's review after standup tomorrow.

brad-advantzware reviewed (changes requested):
Remove N:\ reference

brad-advantzware reviewed (changes requested):
Dev MESSAGE in asiUpdateDBs

Commit:
asiupdate:
cleaned messaging - 577
improved search/run for sub-programs - 1341, 1460
corrected inter-program flicker - 1383
capture connect errors - 1425, 1450

asiUpdateDB.w
enforced close and move on after processing - 504,587
removed diagnostic messages - 1545

asiUpdateENV.w
reformatted several messages (spacing, indention, etc.)
QtyPerSet change was in two updates, removed one - 2179
corrected oe-rel.lot-no EXPORT line - 2226
fixed addon user name - 3500
set success flag to properly report completion - 3608
corrected trigger error in sys-ctrl maintenance caused by FIND trigger - 4306
fixed DB level validation code - 4484
